app/version/release-notes

```diff
@@ -1,2 +1 @@
-When websites ask for camera and mic permissions, you can now use these features on sites you visit.
-We've also improved how you manage the permissions you've already granted, giving you global controls over permissions for individual websites and the ability to restrict whether websites can ask for permissions at all.
\ No newline at end of file
+When websites ask for camera and mic permissions, you can now use these features on sites you visit.
```

Comment:
Ignore this, I'll change them manually because they are too long because we include the hiring bit which we don't need in this release.